src/Sql/Sql/ManagedInstance/Services/AzureSqlManagedInstanceAdapter.cs

```diff
@@ -43,6 +47,33 @@ public class AzureSqlManagedInstanceAdapter
         /// </summary>
         public IAzureContext Context { get; set; }
 
+        /// <summary>
+        /// A private instance of ActiveDirectoryClient
+        /// </summary>
+        private ActiveDirectoryClient _activeDirectoryClient;
+
+        /// <summary>
+        /// Gets or sets the Azure ActiveDirectoryClient instance
+        /// </summary>
+        public ActiveDirectoryClient ActiveDirectoryClient
+        {
+            get
+            {
+                if (_activeDirectoryClient == null)
+                {
+                    _activeDirectoryClient = new ActiveDirectoryClient(Context);
```

Comment:
In the test, it is reported the constructor here canno be reposrted. Please check the code and dependency.

Reply:
tests fixed